.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+This PR...
+
+## Changes
+
+-
+
+## Fixes
+
+-
```

Comment:
Do you guys use this? I usually don't use this format.

Reply:
I usually just add a link to the issue

Reply:
I'll keep them then